warehouse/static/sass/blocks/_notification-bar.scss

```diff
@@ -120,4 +120,38 @@
   &--success {
     background-color: $success-color;
   }
+
+  &__status {
+    &__incidents__message {
+      display: none;
+      &--visible {
+        display: block;
+      }
+    }
+
+    &__components, &__incidents {
```

Comment:
this is an antipattern for BEM, I had the same problem recently and solved it following [this guide](http://getbem.com/faq/#css-nested-elements)